### ISSUE TITLE
error during install

### ISSUE THREAD

**Issue by Morette** [ecosse3/nvim#11]
i've tried to install but get this error

```
Cloning Ecovim configuration... 
Cloning into '/home/thiago/.config/nvim/ecovim'...
warning: Could not find remote branch install_script to clone.
fatal: Remote branch install_script not found in upstream origin
Failed to clone repository. Installation failed.
```

**Comment by ecosse3** [comment]
@Morette Sorry for that, I was experimenting with that and pushed the wrong file. Could you try now?

**Comment by Morette** [comment]
almost, now it freeze here

![image](https://user-images.githubusercontent.com/20028647/144514595-b128c25d-beb7-48a3-9a14-cf2af004982b.png)

**Comment by Morette** [comment]
when I try to open nvim, this erros is show, maybe because the installation is not complete?
![image](https://user-images.githubusercontent.com/20028647/144514948-f9124737-9b5a-449c-b854-2b9cf17e0fbb.png)

**Comment by Morette** [comment]
Another thing, installation proceed in this computer that I'm running arch and have nvim installed.
I have another running ubuntu that I use nvim with appimage and throws an error saying that nvim is not installed.

I can open another issue for this one if you prefer

**Comment by ecosse3** [comment]
@Morette Could you check now? I had to install cmp by default with lazy because there are a lot of dependencies in a project which needs that to work.

If you could open another issue for "nvim" not installed that's better.

**Comment by Morette** [comment]
same problem
![image](https://user-images.githubusercontent.com/20028647/144525340-76085223-cf52-4d7e-9065-5fa08918e79b.png)

**Comment by ecosse3** [comment]
@Morette I am very close to solving this issue. Will update soon.

**Comment by ecosse3** [comment]
@Morette I think it should work pretty fine now!

I am aware just of one issue with cmp-nvim-ultisnips but errors happen only once, then it works properly. I need to get into that more because lazy loading in cmp is pretty complicated.

Please let me know if installer and Ecovim works now 😄

**Comment by Morette** [comment]
it worked.
had to do a clean install, just running the command with the previous broken install did not work

just one comment, when I first open 3 messages showed up, one was to install Treesitter I think, but I could not read it.
If any actions had to be taken after install, this could be in docs too.
😄

**Comment by ecosse3** [comment]
Great! I will add actions to do after installation. I wanted to do them automatically, but somehow it breaks with lazy loading. Maybe in a few days, I will figure it out.
